meshtastic/mesh_interface.py

```
@@ -1742,13 +1752,13 @@ def _queueClaim(self) -> None:
 
     def _sendToRadio(self, toRadio: mesh_pb2.ToRadio) -> None:
         """
-        Send a ToRadio message to the radio device, queuing mesh packets and transmitting non-mesh messages immediately.
+        Send a ToRadio message to the radio transport and manage the interface transmit queue.
 
-        If self.noProto is True the message is not sent. If the provided ToRadio has no `packet` field it is sent immediately via _sendToRadioImpl. If it contains a mesh packet the packet is enqueued (keyed by packet.id) and this method attempts to flush the transmit queue: it waits for free TX queue space, claims queue slots, calls _sendToRadioImpl to transmit queued packets, and re-queues packets that remain unacknowledged. This method may block while waiting for queue space and modifies self.queue.
+        If `toRadio` contains a mesh packet, the packet is enqueued and this method attempts to drain the transmit queue by sending queued packets when transmit slots are available; this operation may block while waiting for free TX space and will re‑queue packets that remain unacknowledged. If `toRadio` contains no mesh packet, it is sent immediately. If protocol handling is disabled on the interface (`self.noProto`), the message is not transmitted.
```

Ruff (0.14.1)

Docstring contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)